django-stubs/contrib/gis/views.pyi

```diff
@@ -1,3 +1,6 @@
-from typing import Any, Optional
+from typing import Dict, Optional, Type
 
-def feed(request: Any, url: Any, feed_dict: Optional[Any] = ...) -> Any: ...
+from django.contrib.gis.feeds import Feed
+from django.http import HttpRequest, HttpResponse
+
+def feed(request: HttpRequest, url: str, feed_dict: Optional[Dict[str, Type[Feed]]] = ...) -> HttpResponse: ...
```

Comment:
Also poorly documented: https://docs.djangoproject.com/en/4.1/ref/contrib/gis/feeds/

Again poked around to determine types: https://github.com/django/django/blob/84206607d6bfd61e7f7a88b51163ffd4153e3b5a/django/contrib/gis/views.py#L23